src/main/java/pl/simpleascoding/tutoringplatform/dto/TeacherDTO.java

```diff
@@ -0,0 +1,7 @@
+package pl.simpleascoding.tutoringplatform.dto;
+
+import org.hibernate.annotations.Immutable;
+
+@Immutable
+public record TeacherDTO(String name, String surname, String email) {
+}
```

Comment:
Do we need to have a separate class for teacher info? Wouldn't it be better to use our existing `UserDTO` class instead?

Reply:
You're right. I thought that we just don't need teacher id, but it may be important to frontend in sending requests